content/docs/tech-insights/scorecards/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Checks
+publishedDate: '2022-11-15'
+description: Managing Scorecards.
+---
+
+# Managing Scorecards
+
+Scorecards are collections of multiple Checks, which may come from different Data Sources.
+
+Overview and manage your Scorecards via Tech Insights → Scorecards page. In this page you can also add Scorecards.
```

Comment:
I'm not sure 'Overview' can be used as a verb like this.
Maybe 'You can see the overview and manage your...'?